roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -8,6 +8,12 @@ assetConfig:
   logoutURL: ""
   masterPublicURL: {{ openshift.master.public_api_url }}
   publicURL: {{ openshift.master.public_console_url }}/
+{% if openshift.master.logging_public_url %}
```

Comment:
Also, there's no need to open a new PR for these changes. Just push new commits into your master branch and the PR will be updated automatically :smile_cat: